dist/redhat/scylla.spec

```diff
@@ -118,6 +120,7 @@ rm -rf $RPM_BUILD_ROOT
 /opt/scylladb/bin/*
 /opt/scylladb/libreloc/*
 /opt/scylladb/libexec/*
+/opt/scylladb/node_exporter/*
```

Comment:
I think it's better to put this in a subpackage. The main package should depend on it, but scylla-server should be installable without it.